README.md

```diff
@@ -2,7 +2,8 @@
 REST API for managing cache control information for pages within the legacy CMS
 
 ### Getting started
-
+* Ensure Docker is installed on your local machine, installation steps can be found here https://docs.docker.com/desktop/install/mac-install/
+* Run `docker run --name mongo-test -p 27017:27017 -d mongo`. This command starts a new container named mongo-test using the official MongoDB image, mapping port 27017 of the host to port 27017 of the container, and runs it in the background.
```

Comment:
This is just a suggestion, so feel free to disagree, but is it worth making this part of `make debug`? That way it all happens with just one command.

Reply:
This is not a bad suggestion, however, we don't normally run backing services via the Makefile as developers might have different configurations for things like mongo. Eg. they may have a local version already running, they may be tunnelling to an env, they may be using docker-compose. For this reason we usually define a stack in dp-compose instead.